01-module-system/helper.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs')
+const chalk = require('chalk');
+function getNotes () {
+    try {
+        const data = fs.readFileSync('notes.json')
+        const content = data.toString()
+        const jsonData = JSON.parse(content)
+        return jsonData
+    }catch(err){
+        console.log(chalk.red("err"))
+       return []
+    }
+    }
+
+const saveNotes = (data)=>{
+    const jsonData = JSON.stringify(data)
+    fs.writeFileSync('notes.json',jsonData)
+}
+const addNotes=(title,body)=>{
+    const jsonData1 = getNotes()
+    const duplicate=jsonData1.find(x=>{if(x.title===title)return true})
```

Comment:
please add whitespaces around `=`, also you can simplify `find`s' callback.
[stackoverflow](https://stackoverflow.com/a/12462414) see first example